runtime/src/main/cpp/Exceptions.h

```diff
@@ -55,10 +55,11 @@ void RUNTIME_NORETURN ThrowNotImplementedError();
 // Throws illegal character conversion exception (used in UTF8/UTF16 conversions).
 void RUNTIME_NORETURN ThrowIllegalCharacterConversionException();
 void RUNTIME_NORETURN ThrowIllegalArgumentException();
+void RUNTIME_NORETURN ThrowIllegalStateException();
```

Comment:
`ThrowIllegalStateException` is not `void`, it returns an object reference.

Reply:
Offline.